### ISSUE TITLE
Test failure in "upload" API endpoint

### ISSUE THREAD

**Issue by schbetsy** [cfpb/hmda-platform#517]
```
[info] - must return 400 when trying to upload to a completed submission *** FAILED ***
[info]   202 Accepted was not equal to 400 Bad Request (InstitutionsHttpApiSpec.scala:146)
```

on Master.

**Comment by jmarin** [comment]
Thanks @schbetsy for reporting this. I think it's related to PR #511, @kgudel can you take a look? For some reason Travis didn't build that PR

**Comment by kgudel** [comment]
This was fixed by PR #522
